.gitignore

```diff
@@ -31,3 +31,5 @@ tmp-project
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+build/
```

Comment:
Is this folder being used somewhere? I don't see anything referencing it

Reply:
It's the default directory for asbuild.  I thought having it would be useful if in the future we move to using its defaults instead of `out/main.wasm`

Reply:
let's add it when we start using it then; I still think the root directory would be the wrong place for such a folder in this project, though